packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx

```diff
@@ -249,8 +255,17 @@ const DaoSettings: React.FC<DaoSettingFormProps> = ({ onClickBack }) => {
         onTransactionFailure: (error) => {
           enqueueSnackbar(error, { variant: 'error' });
         },
-        onTransactionSuccess: () => {},
-        onCompleteAllTransactions: () => {},
+        onTransactionSuccess: (_, receipt: ContractReceipt) => {
+          containerHash = Proposal.getContainerHashFromReceipt(
```

Comment:
@yuetloo 

could you use `proposalInstance` instead of `Proposal`. I made `proposalInstance` as a wrapper for `Proposal` for now.. `proposalInstance` is already created in the same file.